VideoLocker/res/color/edx_segmented_control_background_selector.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:state_checked="true" android:color="@color/edx_grayscale_neutral_light" />
-    <item android:color="@color/edx_grayscale_neutral_white" />
+    <item android:state_checked="false" android:color="@color/edx_grayscale_neutral_white" />
```

Comment:
Can this `selector` be deleted now? It doesn't seem to be used anywhere except as the background to the abstract style `edX.Widget.SegmentedControlSegment`, which is always overridden by the implementations.